db/migrate/20190521200107_update_source_metadata_identifier.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class UpdateSourceMetadataIdentifier < ActiveRecord::Migration[5.1]
+  def change
```

Comment:
Do we do migrations for this? I think we've done rake tasks lately. I personally like this, but I know there's been some difference in opinion. If we keep it like this, this should just be a `def up` with an empty `def down`.